test/product_test.rb

```diff
@@ -35,4 +35,19 @@ def test_update_loaded_variant
     variant.price = "0.50"
     variant.save
   end
+
+  def test_search
```

Comment:
This test isn't very good and doesn't verify that the only products you are getting back from the query are only those of the SKU you were expecting. You just verify that at least the pink iPod was in the result set. I could return all the products and this test would pass because it would eventually find that variant.